sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/BasicWriteStatsTracker.scala

```diff
@@ -192,7 +192,7 @@ class BasicWriteJobStatsTracker(
     new BasicWriteTaskStatsTracker(serializableHadoopConf.value)
   }
 
-  override def processStats(stats: Seq[WriteTaskStats]): Unit = {
+  override def processStats(stats: Seq[WriteTaskStats], duration: Long): Unit = {
```

Comment:
`duration` -> `jobCommitDuration` to make it clear.

Reply:
> `duration` -> `jobCommitDuration` to make it clear.

Done